articles/active-directory/app-proxy/application-proxy-integrate-with-sharepoint-server.md

```diff
@@ -175,7 +175,7 @@ Configure the KCD so that the Azure AD Application Proxy service can delegate us
 To configure the KCD, follow these steps for each connector machine:
 
 1. Sign in to a domain controller as a domain administrator, and then open Active Directory Users and Computers.
-1. Find the computer running the Azure AD Proxy connector. In this example, it's the SharePoint server itself.
+1. Find the computer running the Azure AD Proxy connector. In this example, it's the computer that's running SharePoint Server.
```

Comment:
@kenwithee Is the product SharePoint Server, or should this be "the computer [or server] that's running SharePoint"?

Reply:
I think here it is referring to the computer running.. so this one sounds right to me too.